include/zephyr/net/wifi.h

```diff
@@ -27,6 +27,9 @@
 #define WIFI_LISTEN_INTERVAL_MIN 0
 #define WIFI_LISTEN_INTERVAL_MAX 65535
 
+#define WIFI_RTS_THRESHOLD_MIN 0
+#define WIFI_RTS_THRESHOLD_MAX 2347
+
```

Comment:
You could easily bring CTS as well then, so renaming these to WIFI_RTS_CTR_THRESHOLD_<MIN/MAX>

and adding a set_cts() as well etc...

Reply:
Hi,
The requirement here is emulate support the way RTS threshold being set same as "iw"utility.
As per standard, thresholds are applicable only for RTS frame. There is nothing like CTS threshold. 

If you are mentioning about "selfcts" it is used as protection mechanism as per AP guidance. There is no user setting required.

Reply:
nevermind, there is indeed a unique threshold that applies to the RTS/CTS procedure.